tcms/xmlrpc/api/forms/testplan.py

```diff
@@ -0,0 +1,35 @@
+from django.forms import BooleanField, CharField, ModelChoiceField
```

Comment:
In your commit log:
`Move RPC TestPlan forms to xmlrpc/api/forms.testplan.py` -> `forms/testplan.py` because this is a file path. Alternatively use a module reference but not a mix of both.